examples/proto/gen/typescript/einride/example/freight/v1/index.ts

```diff
@@ -8,25 +8,43 @@ export type Shipment = {
   // The resource name of the shipment.
   name: string | undefined;
   // The creation timestamp of the shipment.
+  //
+  // Behaviors: OUTPUT_ONLY
   createTime: wellKnownTimestamp | undefined;
   // The last update timestamp of the shipment.
   // Updated when create/update/delete operation is shipment.
+  //
+  // Behaviors: OUTPUT_ONLY
   updateTime: wellKnownTimestamp | undefined;
   // The deletion timestamp of the shipment.
+  //
+  // Behaviors: OUTPUT_ONLY
   deleteTime: wellKnownTimestamp | undefined;
   // The resource name of the origin site of the shipment.
   // Format: shippers/{shipper}/sites/{site}
+  //
+  // Behaviors: REQUIRED
   originSite: string | undefined;
   // The resource name of the destination site of the shipment.
   // Format: shippers/{shipper}/sites/{site}
+  //
+  // Behaviors: REQUIRED
```

Comment:
Shouldn't this generate a field which does not contain "undefined" if it is required?

Reply:
The `REQUIRED` behavior means different things depending on where it is used. See https://google.aip.dev/203#required
Specifically when updating a resource, even fields that are marked as `REQUIRED` does not need to be set.

Reply:
Brilliant idea to generate this documentation.